src/lib/Elf.cc

```diff
@@ -36,6 +37,7 @@ Elf::Elf(std::string path, char** imagePointer) {
   char fileMagic[4];
   file.read(fileMagic, 4);
   if (std::memcmp(elfMagic, fileMagic, sizeof(elfMagic))) {
+    std::cout << "[SimEng:Elf]  Elf magic does not match" << std::endl;
```

Comment:
One too many spaces after `]`

Reply:
Is it one space or two spaces as @rahahahat says?